### DISCUSSION TITLE
Incompatible qos policy from studio subscriber

### DISCUSSION THREAD

**Discussion by adrut-cart** [foxglove/community#51] [Q&A]
Hi 
I am getting:
New subscription discovered on topic '/v_dbg', requesting incompatible QoS. No messages will be sent to it. Last incompatible policy: DURABILITY_QOS_POLICY
On ROS2 connection when initiating subscription from studio.

Can I specify QOS policy in panel settings somehow?
Thanks

**Comment by defunctzombie** [comment]
Are you using the rosbridge connection or the ROS 2 native connection in studio?

**Comment by adrut-cart** [comment]
Ros2 native connection in studio

**Comment by bertaveira** [comment]
The issue has been stated in the issues tickets #2473 and foxglove/ros2#92

as far as I am aware it was briefly discussed changing the QoS options and the side effects it would have. Let’s wait for more news about it

**Comment by bertaveira** [comment]
They have just merged a fix for this. It appears to work but at least on my computer I get the error described in issue #2683
